internal/core/services/app.go

```diff
@@ -96,12 +95,10 @@ func (as *AppService) Check(ctx context.Context, dirPath string, deploymentData
 
 		// alert sender is optional
 		if as.alertSender != nil {
```

Comment:
I'd like to suggest to keep `as.alertSender` as always initialized (even with some mock/default values). And use just `alertSender.Send()` in the code. In case when it is enabled, it will process the alert, otherwise just skip it.

Reply:
This code part is deprecated and currently unused, will be removed.